stable/artifactory-ha/values.yaml

```diff
@@ -267,6 +267,8 @@ artifactory:
       path: "artifactory-ha/filestore"
       refreshCredentials: true
       testConnection: false
+      s3AwsVersion: "AWS4-HMAC-SHA256"
```

Comment:
@jainishshah17 - is this ok to have it as a default?

Reply:
No harm in making it configurable. It points to `"AWS4-HMAC-SHA256"` as default if `template="s3"` is used.